src/components/Chat/index.tsx

```diff
@@ -149,6 +150,7 @@ function Chat() {
 
   return (
     <div className="row chatapp-single-chat">
+      <UseScrollToBottom />
```

Comment:
hooks should contain logic only

Reply:
Do you mean the button jsx should be passed as props to the hook?